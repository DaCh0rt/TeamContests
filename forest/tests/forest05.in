14 635
0 3
0 0
1 1
2 0
3 2
3 1
2 3
1 2
5 2
4 0
4 3
5 1
6 0
6 3
