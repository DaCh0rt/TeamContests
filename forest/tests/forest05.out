4007.82 1278216.70
